install_config/install/advanced_install.adoc

```diff
@@ -20,15 +20,18 @@ own implementation using the configuration management tool of your choosing.
 Alternatively, you can use the link:quick_install.html[quick installation]
 method for trial installations.
 
-== Prerequisites
+== Before You Begin
 
-Before installing OpenShift, you must first link:prerequisites.html[satisfy the
-prerequisites] on your hosts, which includes verifying system and environment
-requirements per component type and properly installing and configuring Docker.
+Before installing OpenShift, you must first see the
+link:prerequisites.html[Prerequisites] topic to prepare your hosts, which
+includes verifying system and environment requirements per component type and
+properly installing and configuring Docker. It also includes installing Ansible
+version 1.8.4 or later, as the advanced installation method is based on Ansible
+playbooks and as such requires directly invoking Ansible.
 
 After following the
 instructions in the link:prerequisites.html[Prerequisites] topic, you can
-continue to link:#installing-ansible[installing Ansible].
+continue to link:#installing-ansible[Installing Ansible].
 
 [[installing-ansible]]
```

Comment:
I guess instead of removing it completely, you could just make the whole topic be for Origin only, since the Origin instructions are still valid. Granted, these changes are only going into the enterprise-3.0 branch, so it doesn't really matter / affect the Origin build.